tests/phpunit/Fixtures/SimpleMutationsCollectorVisitor.php

```diff
@@ -43,7 +43,7 @@ public function leaveNode(Node $node)
 
         // It is important to not rely on the keys here. It might otherwise result in some elements
         // being overridden, see https://3v4l.org/JLN73
-        foreach($this->mutator->mutate($node) as $mutatedNode) {
+        foreach ($this->mutator->mutate($node) as $mutatedNode) {
```

Comment:
I thought the CS fixer would pick that up...

Reply:
It doesn't fix fixtures for some reason I didn't yet have time to investigate.